api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -18,6 +18,12 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#protodoc-title: Wasm]
 // [#extension: envoy.bootstrap.wasm]
 
+// Configuration for restricting proxy-wasm capabilities available to modules.
+message CapabilityRestrictionConfig {
+  // The list of proxy-wasm capabilities which will be exposed to the module.
```

Comment:
Can you add a reference to where the capabilities are defined?

Reply:
Added a reference and updated the docs.